### DISCUSSION TITLE
HW1 Part2 Q1

### DISCUSSION THREAD

**Discussion by ewk19001** [cooperrc/computational-mechanics#190] [Q&A]
I am a little confused about part b for this question, for x=0.2*pi this is just one value not a range of values, so what exactly are we supposed to be plotting?

![image](https://github.com/cooperrc/computational-mechanics/assets/135074973/8e46f8fb-5c08-4a53-b5c9-162d74a843dc)

**Comment by kguert** [comment]
I took it as plotting the sine and cosine functions over the range of x values from 0 to 2*pi.

**Reply by ewk19001** [reply]
Thank you, that makes more sense
